docs/api/v4/changes.md

```diff
@@ -1,9 +1,32 @@
 # API v4 Changelog
 
-This page lists additions to the v4 api with each new release of CiviCRM Core.
+*This page lists additions to the v4 api with each new release of CiviCRM Core.*
 
 Also see: [Differences Between Api v3 and v4](/api/v4/differences-with-v3.md).
 
-## Nothing here yet.
+### 5.23 Added PaymentProcessor and PaymentProcessorType APIv4 Entities
 
-API v4 is brand new. No changes yet!
+See https://github.com/civicrm/civicrm-core/pull/15624
+
+### 5.23 $index param supports array input
+
+CiviCRM 5.23 supports two new modes for the `$index` param - associative and non-associative array. See https://github.com/civicrm/civicrm-core/pull/16257
+
+### 5.23 Converts field values to correct data type
+
+The api historically returns everything as a raw string from the query instead of converting it to the correct variable type (bool, int, float). As of CiviCRM 5.23 this is fixed for all DAO-based entities. See https://github.com/civicrm/civicrm-core/pull/16274
```

Comment:
```suggestion
The api historically returns everything as a raw string from the query instead of converting it to the correct variable type (bool, int, float). As of CiviCRM 5.23 this is fixed for all DAO-based entities. See [CiviCRM Core PR #16274](https://github.com/civicrm/civicrm-core/pull/16274)
```

As before...